src/activitylog/activitylog.html

```diff
@@ -53,7 +53,7 @@ <h2>Activity Logs</h2>
           </table>
         </div>
         <div class="log-detail-wrapper" hidden>
-          <div class="log-details"></div>
+          <pre class="log-details"></pre>
```

Comment:
As mentioned in the other comment, let's change this back to a div, the CSS styling should be already enough.